x-pack/plugins/enterprise_search/server/routes/workplace_search/sources.ts

```diff
@@ -219,7 +219,26 @@ export function registerAccountSourceSettingsRoute({
       validate: {
         body: schema.object({
           content_source: schema.object({
-            name: schema.string(),
+            name: schema.maybe(schema.string()),
```

Comment:
Unless I am reading this wrong, this could be DRY'd out and shared below. [Here is an example](https://github.com/elastic/kibana/blob/934dbd8857692be4d10a261741662d9363ce2ee5/x-pack/plugins/enterprise_search/server/routes/workplace_search/sources.ts#L14).

So you could have this:
```JavaScript
const sourceSettingsSchema = schema.object({
  content_source: schema.object({
    name: schema.maybe(schema.string()),
    indexing: schema.maybe(
      schema.object({
        enabled: schema.maybe(schema.boolean()),
        features: schema.maybe(
          schema.object({
            thumbnails: schema.maybe(
              schema.object({
                enabled: schema.boolean(),
              })
            ),
            content_extraction: schema.maybe(
              schema.object({
                enabled: schema.boolean(),
              })
            ),
          })
        ),
      })
    ),
  }),
})
```

and do this in both places:

```JavaScript

      validate: {
        body: sourceSettingsSchema
```

Reply:
Done 👍